src/App.jsx

```diff
@@ -7,55 +7,78 @@ import NewDestinationForm from "./components/NewDestinationForm";
 function App() {
   // eslint-disable-next-line no-use-before-define
   const [destinationsList, setDestinationsList] = useState(destinations);
-  const [input, setInput] = useState({name:"", image:"", continent:"", visited: false, notes:""})
+  const [input, setInput] = useState({
```

Comment:
have a look at https://rjroopal.medium.com/best-practices-for-clean-react-apps-5a891cbe48ab
Point 19 regarding the quote usage ' vs "